tests/arch/arm/arm_irq_vector_table/src/arm_irq_vector_table.c

```diff
@@ -188,33 +179,35 @@ void nrfx_power_clock_irq_handler(void);
 #if defined(CONFIG_BOARD_QEMU_CORTEX_M0)
 void timer0_nrf_isr(void);
 vth __irq_vector_table _irq_vector_table[] = {
-	nrfx_power_clock_irq_handler, 0, 0, 0, 0, 0, 0, 0,
-	timer0_nrf_isr, isr0, isr1, isr2
+	[POWER_CLOCK_IRQn]nrfx_power_clock_irq_handler,
```

Comment:
According to the coding guidelines, when an array is initialized using designated initializers, its size should be specified explicitly (see rule 9.5 [here](https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#id29)).